libr/core/cbin.c

```diff
@@ -2067,6 +2067,7 @@ static int bin_symbols(RCore *r, int mode, ut64 laddr, int va, ut64 at, const ch
 				RFlagItem *fi = r_flag_set (r->flags, fnp, addr, symbol->size);
 				if (fi) {
 					r_flag_item_set_realname (fi, n);
+					fi->demangled = !!sn.demname;
```

Comment:
just a cast should be fine (bool)(size_t)

Reply:
ok f1680776356163b9c30486497b89525a92a744c1